en/docs/reference/connectors/creatio-connector/creatio-connector-example.md

```diff
@@ -0,0 +1,360 @@
+   "FacebookId": "",
+   "LinkedInId": "",
+   "TwitterId": "",
+   "ContactPhoto@odata.mediaEditLink": "Contact(603af7c8-5da7-4a30-bc21-dd290256bb73)/ContactPhoto",
+   "ContactPhoto@odata.mediaReadLink": "Contact(603af7c8-5da7-4a30-bc21-dd290256bb73)/ContactPhoto",
+   "ContactPhoto@odata.mediaContentType": "application/octet-stream",
+   "TwitterAFDAId": "00000000-0000-0000-0000-000000000000",
+   "FacebookAFDAId": "00000000-0000-0000-0000-000000000000",
+   "LinkedInAFDAId": "00000000-0000-0000-0000-000000000000",
+   "PhotoId": "00000000-0000-0000-0000-000000000000",
+   "GPSN": "",
+   "GPSE": "",
+   "Surname": "Test",
+   "GivenName": "API",
+   "MiddleName": "",
+   "Confirmed": true,
+   "IsNonActualEmail": false,
+   "Completeness": 30,
+   "LanguageId": "6ebc31fa-ee6c-48e9-81bf-8003ac03b019",
+   "Age": 19,
+   "MTCMStatusId": "00000000-0000-0000-0000-000000000000",
+   "MTCMSelfCareRoleNewId": "00000000-0000-0000-0000-000000000000",
+   "MTCMInfluencerRatingStr": "",
+   "MTCMBudget": 0,
+   "MTCMTarget": 0,
+   "MTCMRoleId": "00000000-0000-0000-0000-000000000000",
+   "MTSGoNogoId": "00000000-0000-0000-0000-000000000000",
+   "MTCMVerticalId": "00000000-0000-0000-0000-000000000000",
+   "MTCMInfluencerRatingId": "00000000-0000-0000-0000-000000000000"
+}
+
+```
+
+### Get contact
+
+This cURL command retrieves a contact from the Creatio CRM.
+
+**Request**
+
+```curl
+
+curl --location --request GET 'http://mi-hostname:8290/creatioAPITest/contact/603af7c8-5da7-4a30-bc21-dd290256bb73' \
+--header 'Accept: application/json;odata=verbose' \
+--header 'Content-Type: application/json' \
+--header 'ForceUseSession: true' \
+--header 'BPMCSRF: bHZfVOIO4.iSuXwMebGibe' \
+--header 'set-cookie: UserName=83|117|112|101|114|118|105|115|111|114; expires=Sat, 07-May-2022 08:46:12 GMT; path=/; secure; HttpOnly' \
+--header 'Cookie: UserName=83|117|112|101|114|118|105|115|111|114; BPMLOADER=1ndbm4mg3bujyg4syir5gzyw; .ASPXAUTH=DDEF5C8454B7F2DC87B8250B7BB286CCEDB46A5DA6B08BD447E62EC65949BE5C9D37380B095928EB181751D8E6BAB4D8081C3EF7BAF31561BCAF43572BE5E77C7223F411ED9EBED1A369B7835726C464F75FDE56E6D0F2D78BD56671A46561B179A5C93D7D8DB4F04EEB7BF689AA24FB9D348E6E5928318015D45D52CCA9E5B1A597FCE02170EF75235F9B1245042A87F7B0236BF9D39F10235FDEAE0D078C680E0E09362AF91BFCDEDBC0B8CDBBC2858F1652ADAA6CD367C0F9BBBF2F309A2B59E3EC2D310CB457EF17507AEFFE4E6E82F6C1F4994C130E49059408FCC2E3D97ADDF97CB111E9FC9A0E72CC7FD7F560032C56D420FDC898C86E90C406C5A7A27E1CDF65E09502EAD22FFD7DE69EB139E2A3FB13DC5A88ADD894DDD30615D3C5A83012C41CCFECA48C9D34C97EEB4D56FCE13375F78EBFC91A7695DA1335C9EA03AEC108F6EE71D6F8DCEC9E9E0107A4F8A666486D990D809E2F573077FB1736CC29027B0F3B0260DE572B3787046B04170B000B82F10AD8C18231A1BC199D6EC97B436B497F0BCC42C0DF71C2974FBAEC8243B2FB679C7283E27817840FFE19314F07596D95069EDE22741C95B732D426A2F0E08685787FE515AC5795FF684416D81F714B040134F4CD1562C65026D468ADA59AD64E460718A8F465BF780EA897593CCE5627626672BA6BD23D135947BBF7AF8D; BPMCSRF=bHZfVOIO4.iSuXwMebGibe'
```

Comment:
@SanojPunchihewa 

In this case does the new curl look something like this:

```
curl --location --request GET 'http://mi-hostname:8290/creatioAPITest/contact/603af7c8-5da7-4a30-bc21-dd290256bb73' \
```

Reply:
let's leave the `Accept` header as well,

```curl
curl --location --request GET 'http://mi-hostname:8290/creatioAPITest/contact/603af7c8-5da7-4a30-bc21-dd290256bb73' \
--header 'Accept: application/json;odata=verbose'
```